### DISCUSSION TITLE
Recording video length too long

### DISCUSSION THREAD

**Discussion by kuumaur** [blakeblackshear/frigate#3658] [Ask A Question] [answered]
Hey Guys,
I have got an issue which happens every few days.
Some clips/videos are way too long.
30 to 120 minutes
Any ideas why that happens?
- Br Kuumaur

I experimented with these settings, but I am not sure, if these are the correct parameters.
"record": {
        "enabled": true,
        "events": {
          "max_seconds": 60, # Old value=300 New value=60
          "objects": null,
          "post_capture": 10, # Old value=5 new value=10
          "pre_capture": 10, # Old value=5 new value=10
          "required_zones": [
            zone_0
            ],
          "retain": {
            "default": 64,
            "mode": "motion",
            "objects": {}
          }
        },
        "expire_interval": 60,
        "retain": {
          "days": 64,
          "mode": "all"
        },
        "retain_days": null
      },

**Comment by NickM-27** [comment]
Your record mode is `all` so all segments will be apart of that event, not just the ones with motion. 

Need more details, why do you expect the clips to be shorter? Specific example of a long event and what happened?

**Comment by kuumaur** [comment]
Thanks for the quick answer. Well that might be a dump question, but I just started with cctv.
In general the Clips are short, around 30sec., but some get really huge and I dont know why.
I will change the mode from all to motion.

**Reply by NickM-27** [reply]
Are they including the object the entire time though? Frigate will keep recording segments as part of the event as long as the object is visible

**Comment by kuumaur** [comment, marked as answer]
@NickM-27 You are right. I found the victims.
1. My camera has the ability to switch on light, when movement happens, especially during twilight (day to night/night to day). When it switches on the light, the whole picture changes of course, frigate starts identifying objects. After 30sec the light switches of until next movement, etc. -> Solution: Set the permanent night mode in camera.
2. With record mode set to motion the clips are now short or short enough
Thank you
